spec/controllers/api/v1/work_groups_controller_spec.rb

```diff
@@ -204,4 +205,75 @@
       end
     end
   end
+
+  describe '#initiate' do
+    context 'success' do
+      it 'sets the progess status of a work group to in_progress' do
+        patch :initiate, params: { id: work_group2.id, format: :json }
+        expect(controller.instance_variable_get(:@work_group).aasm_state).to eq('in_progress')
```

Comment:
if the controller action ends in `render json:` then in the tests we should check the `JSON.parse(response.body` rather the instance variables